nix/modules/srcCleaned.nix

```diff
@@ -1,20 +1,32 @@
 { self, lib, inputs, config, ... }@flake:
 
 let
+  # Filter out files or folders with this infix
+  matchInfix = infix:
+    root: path: type:
+      lib.strings.hasInfix infix path;
+
   includeCommon =
     [ "crates" "Cargo.toml" "Cargo.lock" "rustfmt.toml" "nextest.toml" ];
```

Comment:
Do we want to include `sql` (holochain_sqlite) and yaml (hc_bundle)?

Reply:
i assume they are included recursively from `"crates"` or else i'd expect failing tests. do we have any indication that they're not included?

Reply:
Ahh right, I think my understanding of how this is working wasn't quite right. No, all good then